ai-platform-unified/notebooks/unofficial/pipelines/google-cloud-pipeline-components_automl_images.ipynb

```diff
@@ -592,16 +592,16 @@
       "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def pipeline(project: str = PROJECT_ID):
</pre><p>You can use a variable as the default?</p><p> </p><p>Didn't know that was possible and even if possible, I haven't seen this used often.</p><p>Consider just removing the default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/341/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
It works well, and is a useful idiom for running pipelines in notebooks.  Hmm, let me give this some thought.